std/v2/algorithm/comparison.d

```diff
@@ -0,0 +1,24 @@
+module std.v2.algorithm.comparison;
+
+// This will be an often-used idiom to fetch the prev version of a module.
+import v1 = std.algorithm.comparison;
+
+/**
+[@@@TODO@@@ Documentation specific for version 2. It should be appended to the default as
```

Comment:
This is relatively easy to do in a doc generator, but most aliases probably don't want this behavior. I think if I was going to do it in the adrdox I'd do it with antoher magic macro like

```
/++
   $(SHOW_ALIAS_DOCS)

   Supplemental stuff here.
+/
```

And the magic macro there is expanded to be a copy of the thing it is an alias of or something like that.

I'm not sure I love it though, just plain linking is probably a better experience more often than not.

Reply:
Yah, I too think a sort of directed choice is best and most flexible.